docs/release-notes/NuGet-6.3.md

```diff
@@ -0,0 +1,125 @@
+
+* PackageDownload multiple versions doesn't work in Visual Studio.  - [#11798](https://github.com/NuGet/Home/issues/11798)
+
+* [Bug]: Visual Studio restore sometimes sets originalTargetFrameworks incorrectly in project.assets.json - [#11795](https://github.com/NuGet/Home/issues/11795)
+
+* [Bug]: NuGet does not retry some HTTP timeouts - [#11779](https://github.com/NuGet/Home/issues/11779)
+
+* [Bug]: misspelling in RestoreCommandCannotDeterminePackagesFolder_deu - [#11774](https://github.com/NuGet/Home/issues/11774)
+
+* Update SPDX licenses to bb0099c - [#11765](https://github.com/NuGet/Home/issues/11765)
+
+* "Illegal characters in path" (Solution Directory) - [#11764](https://github.com/NuGet/Home/issues/11764)
+
+* NuGet Package Manager window causes persistent WPF frame rate spike due to a runaway animation - [#11746](https://github.com/NuGet/Home/issues/11746)
+
+* [Bug]: PM UI version list only shows a single latest version - [#11734](https://github.com/NuGet/Home/issues/11734)
+
+* Large number of allocations while processing package references - [#11733](https://github.com/NuGet/Home/issues/11733)
+
+* Unnecessary Allocations in SemanticVersion.ParseSections() - [#11732](https://github.com/NuGet/Home/issues/11732)
+
+* [Bug]: new warning for package source mappings doesn't pass a value for the resource string placeholder - [#11709](https://github.com/NuGet/Home/issues/11709)
+
+* [Bug]: Central package management breaks no-op restores - [#11696](https://github.com/NuGet/Home/issues/11696)
+
+* [Bug]: MsBuild version is not parsed correctly when -MsBuildPath option is passed to nuget.exe restore - [#11689](https://github.com/NuGet/Home/issues/11689)
+
+* [Bug]: Very slow restore or OOM when using NoWarn - [#11669](https://github.com/NuGet/Home/issues/11669)
+
+* [Bug]: Automatic credential plugin discovery is broken when 64 bit msbuild.exe is used by nuget.exe - [#11623](https://github.com/NuGet/Home/issues/11623)
+
+* [Bug]:  Reduce memory allocation while detecting cycles or potential degrades in package versions during restore - [#11614](https://github.com/NuGet/Home/issues/11614)
+
+* Avoid JTF.Run wrapped property retrieval, use async methods instead. - [#11199](https://github.com/NuGet/Home/issues/11199)
+
+* .nupkg.metadata locked and being used by another process - [#10882](https://github.com/NuGet/Home/issues/10882)
+
+* Unexpected error “Your project file doesn’t list ‘win’ as a “RuntimeIdentifier”” occurs when building the solution after enabling “RestoreLockedMode” - [#10590](https://github.com/NuGet/Home/issues/10590)
+
+* NuGet.exe pack issues a warning (NU5128) when packing a project file - [#8713](https://github.com/NuGet/Home/issues/8713)
+
+* Transitive lock files (with wildcard) result in NU1004 - [#8465](https://github.com/NuGet/Home/issues/8465)
+
+**StillOpens:**
+
+* [Bug]: PM UI Updates tab shows incorrect packages in multitargeting projects - [#11945](https://github.com/NuGet/Home/issues/11945)
```

Comment:
Still opens are supposed to be reviewed and fixed up. 

In this case, the milestone is incorrect.